config/server.lua

```diff
@@ -1,5 +1,6 @@
 return {
     stress = {
         disableForLEO = true, -- If true, it will disable stress for people in the leo job type
+        disableStress = false, -- If true, it will disable stress for everyone
```

Comment:
can we switch this to enableStress = true? I think we should prefer using enableFoo instead of disableFoo
```suggestion
        enableStress = true, -- If false, it will disable stress for everyone
```